gui/wxpython/startup/guiutils.py

```diff
@@ -692,6 +697,58 @@ def delete_grassdb_interactively(guiparent, grassdb):
     return deleted
 
 
+def can_switch_mapset_interactively(guiparent, grassdb, location, mapset):
```

Comment:
Although this is consistent name with the others, the name is little misleading because it reads as "Can I switch mapset interactively?" when in fact it should be saying more "Check interactively if I can switch mapset". I think `can_switch_mapset_interactive()` keeps it simple and it expresses well that it is a "can (I) switch mapset? but doing it in interactive manner.